automem/api/runtime_recall_routes.py

```
@@ -0,0 +1,108 @@
+    if tag_mode not in {"any", "all"}:
+        tag_mode = "any"
+
+    tag_match = (request_obj.args.get("tag_match") or "prefix").strip().lower()
+    if tag_match not in {"exact", "prefix"}:
+        tag_match = "prefix"
+
+    time_start, time_end = parse_time_expression_fn(time_query)
+
+    if start_param:
+        try:
+            time_start = normalize_timestamp_fn(start_param)
```

Ruff (0.15.1)

[error] Local variable `time_start` is assigned to but never used

Remove assignment to unused variable `time_start`

(F841)